src/mutation-observer.ts

```diff
@@ -55,7 +55,7 @@ const processTramTags = (node: Node | TramOneElement) => {
 			const effectReaction = observe(() => {
 				// verify that cleanup is a function before calling it (in case it was a promise)
 				if (typeof cleanup === 'function') cleanup();
-				cleanup = effect();
+				cleanup = effect(node);
```

Comment:
this is what is required for allowing us to accept ref as a parameter for `useEffect` - we pass the node we are processing back into the effect it defined!